flight-website/app/templates/application.hbs

```diff
@@ -1,5 +1,8 @@
 {{page-title "Flight icons"}}
 <DsHeader />
+<div class="ds-beta-banner">
+  <strong>Notice</strong>: there is a new "beta" version of this website:
+  <a href="https://hds-website-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```

Comment:
```suggestion
  <a href="https://hds-website-hashicorp.vercel.app/icons/library" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```
We _could_ link directly to the icons section if we wanted

Reply:
👍